src/KubernetesClient.csproj

```diff
@@ -23,5 +23,6 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
     <PackageReference Include="System.ValueTuple" Version="4.4.0" />
     <PackageReference Include="YamlDotNet.NetCore" Version="1.0.0" />
+    <PackageReference Include="System.Net.WebSockets.Client" Version="4.3.2"/>
```

Comment:
seems netstandard can use websocket without this dependeny

Reply:
I think you can on netstandard2.0 but it doesn't work on `netstandard1.4`. Did you run `dotnet restore` when you tested this?